README.md

```diff
@@ -34,6 +34,7 @@ type Expr
     = And (List Expr)
     | Exact String
     | Exclude Expr
+    | Fuzzy Int String
```

Comment:
Why not `Fuzzy Int Expr` because we can have Fuzzy with an exact terms. Or maybe not.

Reply:
Maybe we can reuse Word type ? `Fuzzy Int Word`

Reply:
I like the explicitness of *Fuzzy*

Reply:
I don't know if we can consider that `Fuzzy` as a End Expr (like Word and Prefix). In my mind, I really like the type `Fuzzy Int Word` or maybe `Fuzzy Int Expr`. @rtxm Do you have a feedback about that?

Reply:
> I don't know if we can consider that Fuzzy as an End Expr (like Word and Prefix)

I don't understand what you're saying. How is `abc~2` different from `abc*` for instance? Also: aren't we nitpicking a bit too much here?

Patch works and tests are passing, I'd suggest we challenge the feature and/or the implementation, but type aliases, I'm not too sure.

Reply:
Ok, maybe you're right.

Reply:
What I understand of the ElasticSearch documentation it that Fuzzy matches are Terminal Expressions: they apply to single strings, not to other expressions or groups. It's very like prefix indeed.

Reply:
Ok Sorry, for my misunderstood @n1k0.